DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -316,5 +316,64 @@ public void Get_section_content_should_return_content_if_only_tutorial_status_is
             );
             result.Should().BeEquivalentTo(expectedSectionContent);
         }
+
+        [Test]
+        public void Get_section_content_should_order_by_tutorial_id_when_shared_order_by_number()
+        {
+            // When
+            const int candidateId = 1;
+            const int customisationId = 8194;
+            const int sectionId = 216;
+
+            // All in section 216
+            // Tutorial: 927  OrderByNumber 34
+            // Tutorial: 928  OrderByNumber 35
+            // Tutorial: 929  OrderByNumber 35
+            // Tutorial: 930  OrderByNumber 36
+            var result = sectionContentService.GetSectionContent(customisationId, candidateId, sectionId);
+
+            // Then
+            var expectedSectionContent = new SectionContent(
```

Comment:
Could maybe use some helper functions to create the section and tutorials here? As we only really care about the order, so only need to specify enough to make sure that's correct 🙂